### ISSUE TITLE
Icons don't show up in svg export

### ISSUE THREAD

**Issue by danielzgtg** [SEG491X-W2023-T42/mikado-machine#147]
The completion icons are missing in the svg export. Furthermore, the accessibility labels are missing.

![image](https://github.com/SEG491X-W2023-T42/mikado-machine/assets/25646384/f44929ed-6189-4b9c-96f4-aac7f6ab490a)
